src/js/utils/styles.js

```diff
@@ -9,6 +9,7 @@ export const baseStyle = css`
   font-size: ${(props) => props.theme.global.font.size};
   line-height: ${(props) => props.theme.global.font.height};
   font-weight: ${(props) => props.theme.global.font.weight};
+  font-variant: ${(props) => props.theme.global.font.variant};
```

Comment:
What happens if there is no `props.theme.global.font.variant`, do we still include this in the CSS? If so, I'm wondering if we should make these more conditional.

Reply:
@jcfilben could you give some inputs?

Reply:
I think we could change these to be mode conditional by doing something like this, where we first check if the prop is defined in the theme before adding the css.
```
${(props) => props.theme.global.font.variant && `
    font-variant: ${props.theme.global.font.variant};
`}
```

Reply:
Strange I'm not sure what is going on in your local environment that is causing the tests to behave oddly. I will go ahead and do the same thing we did last time and update the branch from my end to get the tests to re-run.

If you create any future pull requests it might be worth starting with a fresh fork of grommet to see if that resolves any issues with the tests, but no need to do that for this pull request.

Reply:
@jcfilben noted ! any future PRs will be with a fresh fork! Thanks for taking out the time to update the branch. Let me know if anything else needs to be done for this PR